cherab/openadas/parse/adf21.py

```diff
@@ -19,6 +19,8 @@
 from cherab.core.utility import RecursiveDict
 from .utility import parse_adas2x_rate
 
+CM3TOM3 = 1E-6
```

Comment:
This conversion factor is accesible from the conversion classes:

```
from cherab.core.utility.conversion import Cm3ToM3

Cm3ToM3.conversion_factor
```

Wouldn't it be better to access it this way instead of definig it again?

Reply:
Done